README.md

```diff
@@ -375,11 +401,14 @@ This action deletes versions of a package from [GitHub Packages](https://github.
       token: ${{ secrets.PAT }}
   ```
 
+# GitHub Enterprise Server
```

Comment:
Just adding this section here. Missed adding in the last PR https://github.com/actions/delete-package-versions/pull/91